src/ctype/CTypeUtils.ts

```diff
@@ -8,34 +8,41 @@ import Ajv from 'ajv'
 import { CTypeModel } from './CTypeSchema'
 import ICType from '../types/CType'
 import Crypto from '../crypto'
+import IClaim from '../types/Claim'
 
 export function verifySchemaWithErrors(
-  object: any,
-  schema: any,
+  object: object,
+  schema: object,
   messages?: string[]
 ): boolean {
+  // would like to change these to something more meaningful other then object // don't think we can - verifySchema is called with very different structures
   const ajv = new Ajv()
   ajv.addMetaSchema(CTypeModel)
   const result = ajv.validate(schema, object)
   if (!result && ajv.errors) {
     if (messages) {
-      ajv.errors.forEach((error: any) => {
-        messages.push(error.message)
+      ajv.errors.forEach((error: Ajv.ErrorObject) => {
+        if (typeof error.message === 'string') {
+          messages.push(error.message)
+        }
       })
     }
   }
   return !!result
 }
 
-export function verifySchema(object: any, schema: any): boolean {
+export function verifySchema(object: object, schema: object): boolean {
   return verifySchemaWithErrors(object, schema)
 }
 
-export function verifyClaimStructure(claim: any, schema: any): boolean {
+export function verifyClaimStructure(
+  claimContents: IClaim['contents'], // Checking the SDK and demo-client I feel this should be named claimContents
+  schema: ICType['schema']
+): boolean {
   if (!verifySchema(schema, CTypeModel)) {
     throw new Error('CType does not correspond to schema')
   }
-  return verifySchema(claim, schema)
+  return verifySchema(claimContents, schema)
```

Comment:
I agree this needs to be changed to this. But, it is out of the scope of the ticket.

Reply:
given that the name of the parameter should not have any consequences for the function call, can we not just leave it like this?